commands/preview.js

```diff
@@ -40,7 +40,15 @@ try {
     var siteConfig = require(path.join(path.resolve('./'), '/tests/system/site.js'));
 } catch (e) {
     if (e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
-        console.log('Not using optional site.js.');
+        console.log('Not using optional /tests/system/site.js.');
+    }
+}
+
+try {
+    var siteConfig = require(path.join(path.resolve('./'), '/system/site.js'));
```

Comment:
We moved the tests/system folder from web folder to the root directory for the project which has web and app both, like Matalan, extra, because we want to reuse the web checkout tests. So in this case, the site.js path will be different.